In-house patch to add support for Solaris 11.3 and 12.0.  This patch
has not yet been submitted upstream due to requirements for 3rd party
CI testing.

Removal of duritong/sysctl removes a warning from puppet module list and
is not appropriate for upstream.

diff --git a/metadata.json b/metadata.json
index 0d2958e..fa97bb2 100644
--- a/metadata.json
+++ b/metadata.json
@@ -29,6 +29,14 @@
       "operatingsystemrelease": [
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "Solaris",
+      "operatingsystemrelease": [
+        "12.0",
+        "11.4",
+        "11.3"
+      ]
     }
   ],
   "description": "Installs and configures OpenStack Neutron (Networking).",
@@ -38,7 +46,6 @@
     {"name":"openstack/nova","version_requirement":">=8.2.0 <9.0.0"},
     {"name":"puppetlabs/stdlib","version_requirement":">=4.9.0 <5.0.0"},
     {"name":"openstack/vswitch","version_requirement":">=4.2.0 <5.0.0"},
-    {"name":"duritong/sysctl","version_requirement":">=0.0.1 <1.0.0"},
     {"name":"openstack/openstacklib","version_requirement":">=8.2.0 <9.0.0"}
   ]
 }
